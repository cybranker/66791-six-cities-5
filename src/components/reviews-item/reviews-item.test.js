import React from "react";
import renderer from "react-test-renderer";
import ReviewsItem from "./reviews-item";

const comment = {
  comment: `What an amazing view! The house is stunning and in an amazing location. The large glass wall had an amazing view of the river!`,
  date: `2020-10-25T13:38:44.753Z`,
  id: 1,
  rating: 3,
  user: {
    [`avatar_url`]: ``,
    id: 13,
    [`is_pro`]: false,
    name: `Zak`
  }
};

it(`Should ReviewsItem render correctly`, () => {
  const tree = renderer
    .create(<ReviewsItem
      review={comment}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
