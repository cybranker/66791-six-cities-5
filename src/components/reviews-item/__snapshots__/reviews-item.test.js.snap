// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ReviewsItem render correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src=""
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Zak
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "60%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      What an amazing view! The house is stunning and in an amazing location. The large glass wall had an amazing view of the river!
    </p>
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      October 2020
    </time>
  </div>
</li>
`;
